### PR TITLE
fix(fluentd):  unescape prometheusrules labels

### PR DESCRIPTION
**Pull request by aslafy-z**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| Related tickets | no
| License         | Apache 2.0

### What's in this PR?

This PR updates fluentd prometheusrules summaries and descriptions. 

### Why?

To unescape (wrongly) escaped prometheus labels

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)

### CODE REVIEW

File: pkg/resources/fluentd/prometheusrules.go [hunk 1]
```diff
@@ -158,8 +158,8 @@ func (r *Reconciler) prometheusRules() (runtime.Object, reconciler.DesiredState,
 						"severity":  "warning",
 					},
 					Annotations: map[string]string{
-						"summary":     `Fluentd buffer size prediction warning`,
-						"description": `Fluentd buffer trending watcher`,
+						"summary":     `Fluentd buffer size prediction warning 10 minutes.`,
```
Comment (aslafy-z) [inline pkg/resources/fluentd/prometheusrules.go:161]:
```suggestion
						"summary":     `Fluentd buffer size prediction warning.`,
```

**Review by ahma — APPROVED** [review]
Thanks @aslafy-z lgtm

**Review by tarokkk — APPROVED** [review]

**Comment by aslafy-z** [comment]
@ahma any chance this gets releases any time soon? Thanks you

**Comment by ahma** [comment]
@aslafy-z I hope we can release this in this week, but I can't promise that. Thanks for your patience.

**Pull request merged**